<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgConflictsListDialog</class>
 <widget class="QWidget" name="HgConflictsListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mercurial Conflicts</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="conflictsList">
     <property name="whatsThis">
      <string>&lt;b&gt;Conflicts List&lt;/b&gt;\n&lt;p&gt;This shows a list of files  which had or still have conflicts.&lt;/p&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resolvedButton">
       <property name="toolTip">
        <string>Press to mark the selected entries as 'resolved'</string>
       </property>
       <property name="text">
        <string>Resolved</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unresolvedButton">
       <property name="toolTip">
        <string>Press to mark the selected entries as 'unresolved'</string>
       </property>
       <property name="text">
        <string>Unresolved</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reMergeButton">
       <property name="toolTip">
        <string>Press to re-merge the selected entries</string>
       </property>
       <property name="text">
        <string>Re-Merge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="toolTip">
        <string>Press to edit the selected entry</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="errorGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Errors</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QTextEdit" name="errors">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="inputGroup">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="1" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>327</width>
          <height>29</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="sendButton">
        <property name="toolTip">
         <string>Press to send the input to the hg process</string>
        </property>
        <property name="text">
         <string>&amp;Send</string>
        </property>
        <property name="shortcut">
         <string>Alt+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLineEdit" name="input">
        <property name="toolTip">
         <string>Enter data to be sent to the hg process</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="passwordCheckBox">
        <property name="toolTip">
         <string>Select to switch the input field to password mode</string>
        </property>
        <property name="text">
         <string>&amp;Password Mode</string>
        </property>
        <property name="shortcut">
         <string>Alt+P</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>conflictsList</tabstop>
  <tabstop>resolvedButton</tabstop>
  <tabstop>unresolvedButton</tabstop>
  <tabstop>reMergeButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>errors</tabstop>
  <tabstop>input</tabstop>
  <tabstop>passwordCheckBox</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
